config/webpack/webpack.dev.ts

```diff
@@ -74,7 +79,10 @@ module.exports = (env = {}) =>
                     /([\\/]node_modules[\\/](?!react-native-onyx))/,
                 ],
             },
-        });
+        } as webpack.Configuration);
```

Comment:
```suggestion
        });
```
Not needed

Reply:
Actually it is, property `devServer` causes an error

Reply:
I'm not seeing any error. Try `npm install` perhaps you are missing some types

Reply:
It helped, thanks!